build.gradle

```diff
@@ -51,26 +51,26 @@ ext {
 	googleJsr305Version = '3.0.2'
 	hamcrestVersion = '2.2'
 	hibernateValidationVersion = '8.0.0.Final'
-	jacksonBomVersion = '2.14.2'
-	jaywayJsonPathVersion = '2.7.0'
+	jacksonBomVersion = '2.15.2'
+	jaywayJsonPathVersion = '2.8.0'
 	junit4Version = '4.13.2'
-	junitJupiterVersion = '5.9.2'
-	kotlinCoroutinesVersion = '1.6.4'
-	log4jVersion = '2.19.0'
-	logbackVersion = '1.4.4'
+	junitJupiterVersion = '5.10.0'
+	kotlinCoroutinesVersion = '1.7.3'
```

Comment:
There is also `ext.kotlinVersion = '1.7.0'` in the beginning of this file.
It has to go to: `1.9.10`